src/AudioUnit/AudioComponent.cs

```diff
@@ -38,6 +38,162 @@
 
 namespace XamCore.AudioUnit
 {
+
+#if !COREBUILD
+	// keys are not constants and had to be found in AudioToolbox.framework/Headers/AudioComponent.h
+	[NoWatch, NoTV, Mac (10,13), iOS (11,0)]
+	public partial class ResourceUsageInfo : DictionaryContainer {
+		static NSString userClientK = new NSString ("iokit.user-client");
+		static NSString globalNameK = new NSString ("mach-lookup.global-name");
+		static NSString networkClientK = new NSString ("network.client");
+		static NSString exceptionK = new NSString ("temporary-exception.files.all.read-write");
+
+		public ResourceUsageInfo () : base () {}
+
+		public ResourceUsageInfo (NSDictionary dic) : base (dic) {}
+
+		public string[] IOKitUserClient { 
+			get {
+				var array = GetNativeValue<NSArray> (userClientK);
+				return NSArray.StringArrayFromHandle (array.ClassHandle);
```

Comment:
That should be `.Handle` not `.ClassHandle`
Please complete unit tests so it confirms the value you set can be read back (to the same strings).